src/update-feeds/update-feeds.ts

```diff
@@ -41,6 +44,10 @@ export const startUpdateFeedLoops = async () => {
   );
 };
 
+export type ReadDapiWithIndexResponsesAndChainId = (ReadDapiWithIndexResponse & {
```

Comment:
I don't think you need to do this. If I understand correctly, you only need to pass the `chainId` to the `processBatch` function, which you can do with as a new parameter.